src/games/strategy/triplea/attachments/TriggerAttachment.java

```diff
@@ -2437,7 +2465,8 @@ public static void triggerUnitPlacement(final Set<TriggerAttachment> satisfiedTr
       for (final PlayerID aPlayer : t.getPlayers()) {
         for (final Territory ter : t.getPlacement().keySet()) {
           for (int i = 0; i < eachMultiple; ++i) {
-            // aBridge.getHistoryWriter().startEvent(MyFormatter.attachmentNameToText(t.getName()) + ": " + aPlayer.getName() + " places " +
+            // aBridge.getHistoryWriter().startEvent(MyFormatter.attachmentNameToText(t.getName()) + ": " +
```

Comment:
Dead code, AKA commented out code - should simply be removed so we dont' waste time reading/understanding, formatting it.